syncer.py

```diff
@@ -231,6 +232,46 @@ async def sync_intraday(self, a):
         await series.kv.update(sync_query=a["sync_query"].isoformat())
         a["sync_query"] = a["sync_query"] + timedelta(days=1)
 
+    async def sync_body_data(self, body_data_config):
+        # we fetch multiple body data series with only a single API request.
+        # As a result, for the sync_query, we just use the sync query of the weight
+        sync_query = body_data_config["weight"]["sync_query"]
```

Comment:
Does this make sense? I struggled to understand how to do it properly since this is the first time this plugin creates multiple objects based on one sync/api request.